pkgs/by-name/xp/xpra-html5/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3,
+  uglify-js,
+}:
+stdenv.mkDerivation rec {
+  name = "xpra-html5";
+  version = "17";
+
+  src = fetchFromGitHub {
+    owner = "Xpra-org";
+    repo = "xpra-html5";
+    rev = "v${version}";
+    hash = "sha256-SwP7NazsiUyDD4LUziCwN0X9GTQVq0lYM2jXqNaXLEA=";
+  };
+  buildInputs = [
+    python3
+    uglify-js
+  ];
+  installPhase = "python $src/setup.py install $out /share/xpra/www /share/xpra/www";
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
remove with lib; as it increases eval times